app/src/main/java/acr/browser/lightning/adblock/AdBlocker.kt

```diff
@@ -14,4 +18,7 @@ interface AdBlocker {
      */
     fun isAd(url: String): Boolean
 
+    fun loadScript(uri: Uri): String?
```

Comment:
This should actually be commented, as it's not in use.
It's used for element hiding, or 'cosmetic filtering' as it's called in Yuzu. I haven't really looked at it, but as far as I understand the function is supposed to create a java script that does the element hiding.

Reply:
We could leave this out for now and implement it in the next iteration.

Reply:
is now disabled/commented

I noticed that "element hiding" is called "cosmetic filtering" in uBlock origin. Do you have a preference which name to use?
The hiding/cosmetic is because it only blocks something from being shown, and has no value for privacy.